src/planscape/conditions/load.py

```diff
@@ -75,3 +107,74 @@ def load_metrics(region: str):
                                               condition_score_type=ConditionScoreType.CURRENT, is_raw=metric_is_raw)
                         condition.save()
                         print("Saved Condition: " + condition.raster_name)
+
+
+def compute_elements(region: str, reload: bool):
+    """Computes element rasters based on the conditions config and saves them locally. Optionally loads them to our database.
+
+    Args:
+      region: The region to compute
+      reload: True if the raster should be reloaded 
+
+    """
+    config_path = os.path.join(
+        PLANSCAPE_ROOT_DIRECTORY, 'src/planscape/config/conditions.json')
+    config = PillarConfig(config_path)
+
+    region = config.get_region(region)
+    for pillar in config.get_pillars(region):
+        if pillar['display']:
```

Comment:
Nit: change this to

  if not pillar['display']:
    continue
  ...

It'll be slightly easier to read without the identation